docs/DeveloperGuide.md

```diff
@@ -269,30 +269,299 @@ b. Require developers to download those libraries manually (this creates extra w
 
 Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (unlikely to have) - `*`
 
-
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
-`* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
+`* * *`| user | add a task | record tasks that need to be done �some day�. 
+`* * *`| user |find upcoming tasks | decide what needs to be done soon. 
+`* * *`| user | delete a task | get rid of tasks that I no longer care to track. 
+`* * *`| new user | view more information about a particular command, so that I can learn how to use various commands 
+`* *`| advanced user |  use shorter versions of a command | that can type a command faster.
+`* *`|user | set reminder for my task | can complete the task on time. 
+`* *` |user | view task that is due today when I launch the application | verify what is done and what is to be done. 
+`* *` |user |add category to a task | view task in a more organise manner. 
+`* *` | user | edit my task details, if some changes is needed | So that I can update the latest details.
+`* *`| user | set priority to each of my task | prioritise which task to focus on. 
+`* *`| user | add duplicate tasks for specfic period of time | see a duplicate appears on regular intervals, e.g. once per week. 
+`*`| user | color code a task | group my task easily
+
+
 
```

Comment:
Nice having some more interesting features. Do remember to prioritize the completion of the "must-have" features first.